app/controllers/questions_controller.rb

```diff
@@ -4,13 +4,25 @@ def index
   end
 
   def create
+    logger.debug "PARAMS for create: #{params}"
```

Comment:
I think the Rails default server logs give you the params when a method is called. E.g.:
```
Processing by QuestionsController#create as JSON
  Parameters: {"id"=>nil, "created_at"=>nil, "updated_at"=>nil, "text"=>"new one", "user_id"=>5, "answers"=>[{"id"=>nil, "created_at"=>nil, "updated_at"=>nil, "question_id"=>nil, "text"=>"ok"}], "question"=>{"id"=>nil, "created_at"=>nil, "updated_at"=>nil, "text"=>"new one", "user_id"=>5}}
```